action/actctx.go

```diff
@@ -220,6 +208,19 @@ func (act *AbstractAction) convertToTx() TxCommonInternal {
 			blob:       act.blobData,
 		}
 	default:
-		panic(fmt.Sprintf("unsupported action version = %d", act.version))
+		panic(fmt.Sprintf("unsupported action type = %d", act.txType))
 	}
 }
+
+func (act *AbstractAction) validateTx() error {
```

Comment:
They are redundant validates, already checked when convert to envelope or envelope's sanitycheck

Reply:
this is the validate for `EnvelopeBuilder` now, once we change builder into tx-specific builder, this won't be needed